backend/fms_core/resources/sample.py

```diff
@@ -278,6 +287,11 @@ def after_save_instance(self, instance, using_transactions, dry_run):
         super().after_save_instance(instance, using_transactions, dry_run)
         reversion.set_comment("Imported samples from template.")
 
+    def save_m2m(self, obj, data, using_transactions, dry_run):
+        sampleByProject = SampleByProject.objects.create(project=self.project, sample=obj)
+        data["Project"] = self.project.id
```

Comment:
This is not useful unless you want import_export to try to create a second copy of your sampleByProject when the super() method is called.